### DISCUSSION TITLE
Can't activate extension behind proxy in VSCode

### DISCUSSION THREAD

**Discussion by ajcates** [community/community#11630] [Copilot Conversations] [answered]
Everytime I try and activate the extension I get the following error, I'm guessing that it has todo with the fact that I am behind a http proxy.

`Extension` activation failed: "getaddrinfo ENOTFOUND api.github.com"

**Comment by davecheney** [comment, marked as answer]
I am sorry, currently the Copilot VSCode extension does not support proxies.

**Reply by andrewpolemeni** [reply]
Will it ever support proxies? It's unusable at every company I've worked at.

**Reply by laduran** [reply]
Hmm. This is a bummer. I bought it under my personal account for use with work.
Would be very nice to make co-pilot work with a proxy.

**Reply by laduran** [reply]
OK this may not be as bad as it seems... Seems only initial activation needed a proxy.
Once I logged off the VPN  Co-pilot activated and I was then able to use Co-pilot while behind a proxy.

**Reply by andrewpolemeni** [reply]
However, in my case, I did log off the VPN and then log back on and did authenticate. It only works when I log off the VPN.

**Reply by bartzalewski** [reply]
In my case, it's probably because of the random unavailability of iCloud Private Relay. I've always used it before I started using GitHub Copilot when it launched. No router configurations, no updates, not anything from my side and now I can't use GitHub Copilot even with the Private Relay turned off.
